coap-gateway/service/signIn.go

```diff
@@ -316,6 +316,9 @@ func signInPostHandler(req *mux.Message, client *Client, signIn CoapSignInReq) {
 	} else {
 		client.server.expirationClientCache.Set(deviceID, client, time.Second*time.Duration(expiresIn))
 	}
+
+	client.exchangeCache.Clear()
+
```

Comment:
we want to clear refresh token after success sign in - `client.refreshCache.Clear()`